src/register-org/components/regulatory-organisation-type/regulatory-organisation-type.component.ts

```diff
@@ -121,12 +130,21 @@ export class RegulatoryOrganisationTypeComponent extends RegisterComponent imple
 
   public onContinueClicked(): void {
     if (this.validateForm()) {
-      this.registrationData.regulatoryOrganisationTypes = this.regulators.value as RegulatoryOrganisationType[];
+      // Set corresponding registration data
+      this.registrationData.regulators = this.regulators.value as Regulator[];
       // eslint-disable-next-line @typescript-eslint/no-empty-function
-      this.router.navigate(['/register-org-new/organisation-services-access']).then(() => {});
+      this.router.navigate(['/register-org-new/organisation-services-access']);
```

Comment:
The new path should be constructed using service registerOrgService.REGISTER_ORG_NEW_ROUTE

Reply:
I have updated the code now.